tests/conftest.py

```
@@ -823,6 +823,24 @@ def cleanup_asyncmock_objects(request):
             gc.collect()
 
 
+@pytest.fixture(autouse=True)
+def isolate_db_manager_state():
+    """
+    Reset db_utils manager/cache around every test to avoid leaked SQLite handles.
+    """
+    import mmrelay.db_utils as db_utils
```

Ruff (0.15.10)

[warning] Use `from mmrelay import db_utils` in lieu of alias

Replace with `from mmrelay import db_utils`

(PLR0402)